recipes/pyflct/LICENSE.txt

```diff
@@ -0,0 +1,502 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
you don't need to manually package the license file since it's included in the source distribution.

Reply:
I'll remove it.

Reply:
Fixed.